invenio_rdm_records/services/schemas/__init__.py

```diff
@@ -88,14 +86,18 @@ class Meta:
     @validates("pids")
     def validate_pids(self, value):
         """Validates the keys of the pids are supported providers."""
-        # #PIDS-FIXME is this needed? Otherwise it will fail at sevice
-        # component level when processing. If needed add list of supported
-        # be careful on format not to explode the translation file
-
-        not_allowed_types = set(value.keys()) - self.PIDS_TYPES
-        if not_allowed_types:
-            raise ValidationError(
-                    _("Unkown PID type"),
+        for scheme, pid_attrs in value.items():
+            id_schema = IdentifierSchema(allow_all=True)
+            id_schema.load({
+                "scheme": scheme,
+                "identifier": pid_attrs.get("identifier")
+            })
+
+            is_provider_external = pid_attrs.get("provider") == "external"
```

Comment:
for the next PR: I would put this as a "const" `PROVIDER_EXTERNAL = "external" in the service so it can be re-used.

Reply:
Fixed, thanks!